### PR TITLE
Remove clutter from logger

### PR DESCRIPTION
**Pull request by RafaelTaranto**

### CODE REVIEW

**Comment by RafaelTaranto** [comment]
[Ticker API errors result in log dump of code and CloudFlare HTML](https://trello.com/c/i2KsN9OG/687-ticker-api-errors-result-in-log-dump-of-code-and-cloudflare-html)

**Comment by RafaelTaranto** [comment]
Milestone: 7.3.x Crafty Chnemu

File: lib/logger.js [hunk 1]
```diff
@@ -4,8 +4,12 @@ const options = require('./options')
 const logger = new winston.Logger({
   level: options.logLevel,
   transports: [
-    new (winston.transports.Console)({timestamp: true, colorize: true})
-  ]
+    new (winston.transports.Console)({ timestamp: true, colorize: true })
+  ],
+  rewriters: [(_, __, meta) => {
```
Comment (joshmh) [inline lib/logger.js]: Here's a way to not require dummy variables:

```suggestion
  rewriters: [(...[,,meta]) => {
```

File: lib/logger.js [hunk 2]
```diff
@@ -4,8 +4,12 @@ const options = require('./options')
...
+    if (meta) return meta.toString()
```
Comment (joshmh) [inline lib/logger.js]: Instead of checking truthiness of ``meta``, check if it has ``toString()``.

**Review by joshmh — CHANGES_REQUESTED** [review]

File: lib/logger.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
   level: options.logLevel,
   transports: [
     new (winston.transports.Console)({ timestamp: true, colorize: true })
   ],
-  rewriters: [(_, __, meta) => {
-    if (meta) return meta.toString()
-  }]
-
+  rewriters: [
+    (...[,, meta]) => _.hasIn('toString', meta) ? meta.toString() : meta
```
Comment (joshmh) [inline lib/logger.js]: Hmm, if meta doesn't have a ``toString`` function, we probably don't want to print it out. Maybe this should just be some fixed message, like "Error, no further information is available".

**Review by joshmh — CHANGES_REQUESTED** [review]

File: lib/logger.js (hunk 3 above)
Comment (RafaelTaranto) [reply lib/logger.js]: Good catch! What a brain fart =p

**Pull request merged**